Update readme.md (#16)
clerkinc/clerk-nextjs-starter: README.md

# Clerk - Next.js starter repository

> This repository shows an example use case for how you can authenticate your users using [Clerk](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) as an authentication provider on your Next.js application.

Reviewer: ```suggestion
This repository shows an example about how to add user authenticating to your Next.js application using [Clerk](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

## Running the starter

You have two option to run the starter, we recommend the one click deployment with Vercel. This way, environment variables are set up for you automatically and it's a much easier experience.

### Option 1 - One click deployment with Vercel

> You can use the following deployment button from Vercel, [this article on our blog](https://clerk.dev/blog/clerk-vercel-one-click-deployment?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter) guides you step to step on how to do so.

Reviewer: ```suggestion
You can use the following deployment button from Vercel. For more detailed read our [official step by step guide](https://clerk.dev/blog/clerk-vercel-one-click-deployment?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/clerkinc/clerk-nextjs-starter&integration-ids=oac_7uYNbc9CdDAZmNqbt3LEkO3a&external-id=starter)

### Option 2 - Run manually

To run the example locally you need to:

1. Click [**"Use this template"**](https://github.com/clerkinc/clerk-nextjs-starter/generate) to create a copy of this repository on your GitHub account.

> 2. Clone your repository locally.

Reviewer: ```suggestion
2. Clone your repository.
```

> 3. Sign up for a Clerk account at [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).

Reviewer: ```suggestion
3. Sign up at [Clerk.dev](https://www.clerk.dev/?utm_source=github&utm_medium=starter_repos&utm_campaign=nextjs_starter).
```

> 4. Setup the required Clerk Frontend API variable from your Clerk project as shown at [the example env file](./.env.local.sample).

Reviewer: We should add a step to describe that we need to create an instance.

Author: Added a step!
